Mayhemfile

```diff
@@ -0,0 +1,7 @@
+project: word-checker
```

Comment:
Can we rename this target to indicate what's being fuzzed?

Reply:
I've renamed it